R/repair-names.R

```diff
@@ -172,12 +172,14 @@ repaired_names <- function(name,
   new_name
 }
 
-minimal_names <- function(name, n) {
+minimal_names <- function(name, n = NULL) {
```

Comment:
I had deliberately declined to set a default value here. But perhaps it's fine to use `NULL` as well. But then the check just below should change from `missing(n)` to `is.null(n)`.